src/timeMachine/components/RawFluxDataTable.tsx

```diff
@@ -27,25 +32,67 @@ interface State {
 class RawFluxDataTable extends PureComponent<Props, State> {
   public state = {scrollLeft: 0, scrollTop: 0}
 
+  // TODO(ariel): remove all this when flowsUiPagination is lifted
   private parseFilesWithFromFlux = memoizeOne(parseFilesWithFromFlux)
   private parseFiles = memoizeOne(parseFiles)
   private parseFilesWithObjects = memoizeOne(parseFilesWithObjects)
 
   public render() {
-    const {width, height, files, disableVerticalScrolling} = this.props
+    const {
+      width,
+      height,
+      files,
+      disableVerticalScrolling,
+      startRow,
+    } = this.props
+    let {data, maxColumnCount} = this.props
 
     const {scrollTop, scrollLeft} = this.state
+
+    const tableWidth = width
+    const tableHeight = height
+
+    if (isFlagEnabled('flowsUiPagination')) {
```

Comment:
i can't put my finger on it, but triggering this off of a flag instead of by props feels wrong. what about those other flags below? why aren't they included in this solution?